lib/kafka.ts

```diff
@@ -58,3 +73,92 @@ export const waitForKafkaReady = async function (page: Page, name: string) {
     timeout: config.kafkaInstanceCreationTimeout
   });
 };
+
+export const getBootstrapUrl = async function (page: Page, name: string) {
+  await navigateToKafkaList(page);
+  const instanceLinkSelector = page.getByText(name);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+  await row.locator('[aria-label="Actions"]').click();
+
+  await page.locator('button', { hasText: 'Connection' }).click();
+
+  await expect(page.locator('[aria-label="Bootstrap server"]')).toHaveCount(1);
+  const bootstrap = await page.locator('[aria-label="Bootstrap server"]').inputValue();
+  await page.locator('[aria-label="Close drawer panel"]').click();
+
+  return bootstrap;
+};
+
+// TODO - we shouldn't use just prefix for topic/group but also complete name
+// TODO - we should click on topic name/prefix when it popups when filling the prefix/name
+export const grantProducerAccess = async function (page: Page, saId: string, topicName: string) {
+  await page.getByTestId('actionManagePermissions').click();
+  await page.getByRole('button', { name: 'Options menu' }).click();
+  await page.getByRole('option').filter({ hasText: saId }).click();
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.getByTestId('permissions-dropdown-toggle').click();
+  // TODO - This is another option which should be tested
+  // await page.getByRole('button', { name: 'Add permission' }).click();
+
+  await page.locator('button', { hasText: 'Produce to a topic' }).click();
+
+  await page.getByPlaceholder('Enter prefix').click();
+  await page.getByPlaceholder('Enter prefix').fill(topicName);
+  // TODO - This is just a workaround - create issue for `save` button which is disabled even if the prefix is written
```

Comment:
I don't think this is an issue. When you type, the whispering will offer you a topic name. In case you click on it or anywhere else, the button is working fine. Only when you type in the field is it disabled till the next action. Well maybe it's an issue. So should I open the issue wdyt?